source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst

```diff
@@ -367,6 +368,9 @@ Fields:
 * (=) topologyVersion: A topologyVersion or null. Default null.
   The "topologyVersion" from the server's most recent ismaster response or
   `State Change Error`_.
+* (=) iscryptd: boolean indicating if the server is a
+  `mongocryptd <../client-side-encryption/client-side-encryption.rst#mongocryptd>`_
+  server. Default null.
```

Comment:
IMO, `false` makes more sense as a default for drivers that would model this as a basic boolean (and not a boolean object that can also be null). I realize the CSOT spec only cares about this being `true` or not.

Side question: is this inferred from processing the isMaster/hello response, or is it something SDAM handles differently because it already knows it's talking to a mongocryptd? AFAICT, this is the only mention of `iscryptd` in the PR apart from CSOT talking about reading its value.

Reply:
This is inferred from isMaster/hello responses. I used "null" as the default to explicitly indicate that it may not be present in the response.